platform/windows/display_server_windows.cpp

```diff
@@ -607,8 +607,10 @@ void DisplayServerWindows::show_window(WindowID p_id) {
 		_update_window_style(p_id);
 	}
 
-	ShowWindow(wd.hWnd, (wd.no_focus || wd.is_popup) ? SW_SHOWNOACTIVATE : SW_SHOW); // Show the window.
-	if (!wd.no_focus && !wd.is_popup) {
+	if (wd.no_focus || wd.is_popup) {
+		ShowWindow(wd.hWnd, SW_SHOWNA); // Show the window.
+	} else {
+		ShowWindow(wd.hWnd, SW_SHOW); // Show the window.
```

Comment:
I know the comment existed already but `// Show the window.` to document what `ShowWindow()` does is not so useful.
Clarifying what `NA` means would be relevant though.

Reply:
It is a windows API constant.   I believe anyone working in this area of the code will look it up in the MS references if they need to?

Reply:
added a link to the API docs

Reply:
added URL.  MS reference URLs are usually good for several years